### ISSUE TITLE
Drag to home screen not working

### ISSUE THREAD

**Issue by fungs** [stanipintjuk/Silverfish#59]
It seems that the home screen does not show when dragging the icon to the right display edge on my phone running Android 7.1

**Comment by Lonami** [comment]
Yes, it still doesn't work on Nougat and above, I made a workaround on #53.

**Comment by fungs** [comment]
The workaround does not work for me (using v0.5.1 from fdroid).

**Comment by Lonami** [comment]
That's strange then, perhaps the [threshold](https://github.com/stanipintjuk/Silverfish/commit/58794f6b3a7a46d767f6a0b2f26d1f5247e826d1#diff-62dd99dfd6f341d5150b49a7970632c1R33) is too low?

**Comment by fungs** [comment]
Don't know, I've tried many times with and without moving the icon and it never worked for me.

**Comment by frozencemetery** [comment]
Also on Android 7, I can't drag the icons either.  Additionally, the workaround menu doesn't show up.
